goaviatrix/edge_spoke.go

```diff
@@ -56,6 +56,7 @@ type EdgeSpoke struct {
 	VlanList                           []*EdgeSpokeVlan
 	Vlan                               string                        `json:"vlan,omitempty"`
 	CustomInterfaceMapping             map[string]CustomInterfaceMap `json:"custom_interface_mapping,omitempty"`
+	AdvertisedSpokeRoutes              []string                      `json:"advertise_cidr_list,omitempty"`
```

Comment:
For the API layer, suggest you keep the name of the tag and the field the same. Just to make things a bit more obvious.